api/v3/Job/Iatsrecurringcontributions.php

```diff
@@ -124,7 +124,6 @@ function civicrm_api3_job_Iatsrecurringcontributions($params) {
     // CRM_Core_Error::debug_var('Contribution Template', $contribution_template);
     // generate my invoice id like CiviCRM does
     $hash = md5(uniqid(rand(), TRUE));
-    $failure_count    = $recurringContribution['failure_count'];
```

Comment:
Defining this here pollutes my namespace and is confusing with the params['failure_count'] - I'll just grab it when I need it later.